lumicks/pylake/kymo.py

```diff
@@ -68,6 +68,10 @@ def __getitem__(self, item):
     def _get_photon_count(self, name):
         return getattr(self.file, f"{name}_photon_count".lower())[self.start:self.stop]
 
+    def _get_axis_metadata(self, axis=0):
+        [return_axis] = (cur_axis for cur_axis in self.json["scan volume"]["scan axes"] if cur_axis["axis"] == axis)
```

Comment:
This is true. That function would be nice to have in the codebase.

The thing that I wrote has one advantage over that code though, which is that it throws an error if there's a duplicate axis, while first() from that stackoverflow answer doesn't.

If you're not worried that this will ever be an issue, then I would be happy to change it to first.

Reply:
You can also wrap your code in a function. The function is the main thing that adds clarity.